packages/core/models/notices/WalletNoticeModel.js

```diff
@@ -44,8 +42,11 @@ export default class WalletNoticeModel extends abstractNoticeModel({
   }
 
   message () {
-    return I18n.t(this.get('action'), {
-      address: this.get('address'),
-    })
+    return [
```

Comment:
In some place message returns {value: ''}, but here we have an array... Looks suspicious.